ad7768-1fmcz/ad77681.c

```diff
@@ -351,14 +354,17 @@ int32_t ad77681_set_power_mode(struct ad77681_dev *dev,
 int32_t ad77681_set_mclk_div(struct ad77681_dev *dev,
 			     enum ad77681_mclk_div clk_div)
 {
-	ad77681_spi_write_mask(dev,
-			       AD77681_REG_POWER_CLOCK,
-			       AD77681_POWER_CLK_MCLK_DIV_MSK,
-			       AD77681_POWER_CLK_MCLK_DIV(clk_div));
+	int32_t ret;
+
+	ret = ad77681_spi_write_mask(dev,
+				     AD77681_REG_POWER_CLOCK,
+				     AD77681_POWER_CLK_MCLK_DIV_MSK,
+				     AD77681_POWER_CLK_MCLK_DIV(clk_div));
 
-	dev->mclk_div = clk_div;
+	if (ret==SUCCESS)
```

Comment:
Add some spaces, like this: `if (ret == SUCCESS)`

Reply:
Done fixing